ampd/src/handlers/evm_verify_worker_set.rs

```diff
@@ -0,0 +1,124 @@
+use error_stack::ResultExt;
+use serde::Deserialize;
+
+use axelar_wasm_std::voting::PollID;
+use events_derive::try_from;
+
+use crate::types::{EVMAddress, Hash, TMAddress, U256};
+
+#[derive(Deserialize, Debug)]
+pub struct Operators {
+    pub weights: Vec<(EVMAddress, U256)>,
+    pub threshold: U256,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct WorkerSet {
+    pub tx_id: Hash,
+    pub log_index: u64,
+    pub operators: Operators,
+}
+
+#[derive(Deserialize, Debug)]
+#[try_from("wasm-worker_set_poll_started")]
+#[allow(dead_code)]
+struct PollStartedEvent {
+    #[serde(rename = "_contract_address")]
+    contract_address: TMAddress,
+    worker_set: WorkerSet,
+    poll_id: PollID,
+    source_chain: connection_router::types::ChainName,
+    source_gateway_address: EVMAddress,
+    confirmation_height: u64,
+    expires_at: u64,
+    participants: Vec<TMAddress>,
+}
```

Comment:
do we need a completely separate event?

Reply:
Maybe we can unify with an enum later, but I'd prefer going with everything separate and then come back to refactor if possible.

Reply:
ok